recipes/lycon/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "lycon" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 30515e75d9cbd1ab68f432675c64aa7b955eaccb95987f0128cedcf5272afea4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=2.8
+  host:
+    - python
+    - pip
+    - numpy
```

Try adding this temporarily:
```suggestion
    # TODO: Remove the version constraint and re-render once this is merged into a feedstock
    - numpy 1.16
```

---

Thanks for stepping in @chrisburr, sadly it seems like "downgrading" `numpy` did not help... Any other idea?

---

Linux is now working isn't it?

---

oh yes, sorry, you're right, I rushed my conclusion.... let me check the CI

---

so, OSX seems to fail the import test silently, whereas on windows cmake complains that the PYTHON_LIB_PATH variable is not found/set